### PR TITLE
added changes and links to footer

### PR DESCRIPTION
**Pull request by mvic89**
Just added the right file directory for the logo-image and made it link to the homepage on all pages.

### CODE REVIEW

File: mammals.html [hunk 1]
```diff
@@ -30,7 +30,7 @@
     <footer>
         <div class="footer-container">
             <div class="logo-div">
-                <img class="brand-logo" src="./img/capyzoo-logo.png" alt="capyzoo-logo">
+                <a href="index.html"><img class="brand-logo" src="./assets/footer/capyzoo-logo.png" alt="capyzoo-logo"></a>
```
Comment (maysunktong) [inline mammals.html:33]: Looks good to me!

Comment (Bor-ghol) [reply mammals.html:33]: Looks good to me too :))

**Review by Bor-ghol — APPROVED** [review]
Looks good to me!

**Review by Endorze — APPROVED** [review]
Merge merge merge!

**Pull request merged**